src/components/00-materials/styles/story.js

```diff
@@ -139,7 +117,7 @@ storiesOf('Materials', module)
           width: 250px;
           height: 50px;
           border-radius: 5px;
-          border: 1px solid lightgray;
+          border: 1px solid lightgrey;
```

Comment:
Isn't `lightgray` and `lightgrey` the same colorcode?

Reply:
yes but I decide to name everything "grey" for consistency.

Reply:
american vs english grey